packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -251,7 +251,7 @@ Autocomplete.propTypes = {
   emptyListMessage: PropTypes.string,
   noMatchesMessage: PropTypes.string,
   willClearQueryOnClose: PropTypes.bool,
-  isFullWidth: PropTypes.bool,
+  dropdownProps: PropTypes.any,
```

Comment:
`{ bar: PropTypes.object }` isn't type compatible to `type fooProps = { bar?: barProps }`